src/sentry/integrations/vsts_extension/integration.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from django.contrib import messages
+from django.http import HttpResponseRedirect
+
+from sentry.integrations.vsts.integration import VstsIntegrationProvider
+from sentry.pipeline import PipelineView
+from sentry.utils.http import absolute_uri
+
+
+class VstsExtensionIntegrationProvider(VstsIntegrationProvider):
+    key = 'vsts-extension'
+
+    def get_pipeline_views(self):
+        views = super(VstsExtensionIntegrationProvider, self).get_pipeline_views()
+        views[-1] = VstsExtensionFinishedView()
+        return views
+
+    def build_integration(self, state):
+        # Normally this is saved into the ``identity`` state, but for some
+        # reason it gets wiped out in the NestedPipeline. Instead, we'll store
+        # it in it's own key (``vsts``) to be used down the line in
+        # ``VSTSOrganizationSelectionView``.
+        state['account'] = {
+            'AccountId': state['vsts']['AccountId'],
+            'AccountName': state['vsts']['AccountName'],
+        }
+
+        state['instance'] = '{}.visualstudio.com'.format(
+            state['vsts']['AccountName']
+        )
+
+        return super(
+            VstsExtensionIntegrationProvider,
+            self
+        ).build_integration(state)
+
+
+class VstsExtensionFinishedView(PipelineView):
+    def dispatch(self, request, pipeline):
+        pipeline.finish_pipeline()
+
+        messages.add_message(request, messages.SUCCESS, 'VSTS Extension installed.')
+
+        # TODO: replace with whatever we decide the finish step is.
```

Comment:
Maybe a question for @saragilford

Reply:
Will need some context here

Reply:
I ended up just sending the user to the integration's page, which is probably what we want anyway.

Reply:
@saragilford Basically after the whole VSTS Marketplace Extension installation process, where should we send the user within Sentry. Currently it sends them to that specific Integration's page, which I think makes sense?

Reply:
works for me!